CRM/Member/Form/Membership.php

```diff
@@ -338,10 +334,7 @@ public function setDefaultValues() {
         $defaults['source'] = $defaults['membership_source'];
       }
     }
-    //CRM-13420
-    if (empty($defaults['payment_instrument_id'])) {
-      $defaults['payment_instrument_id'] = key(CRM_Core_OptionGroup::values('payment_instrument', FALSE, FALSE, FALSE, 'AND is_default = 1'));
-    }
+    CRM_Contribute_Form_PaymentDetails::setDefaultValues($defaults, NULL);
```

Comment:
Ah, ok - it's an extraction, which is a good thing, just hard to review the extraction with so many other chagnes